crates/holochain_state/Cargo.toml

```diff
@@ -57,11 +57,14 @@ holochain_state = { path = ".", features = ["test_utils"] }
 anyhow = "1.0"
 arbitrary = "1.0"
 fixt = { path = "../fixt" }
+kitsune_p2p_timestamp = { version = "^0.4.0-dev.2", path = "../kitsune_p2p/timestamp", features = [
+  "full",
```

Comment:
the timestamp has "features"!

Reply:
Off the top of my head, that helps the type be used in wasm by excluding code that can't run in a wasm environment. Couldn't tell you what the other features are for without looking :)